docs/en/observability/monitor-infra/aws-metrics.asciidoc

```diff
@@ -1,6 +1,11 @@
 [[aws-metrics]]
 = AWS metrics
 
+//TODO: Rewrite this to match new format. Requires input from dev team.
+
+//REVIEWERS: This was taken from the existing docs and moved here. Do we want
```

Comment:
Reviewers: Please respond to this question. What do we want to do with the existing docs about non-host metrics? Do we want to try to update them, or just leave as-is for now?

Reply:
Hey @dedemorton, let's leave them as they are. I believe they are still accurate.